packages/plugin-user/test/integration/spec/user.js

```diff
@@ -33,6 +34,56 @@ describe(`plugin-user`, function() {
       return spark.device.register();
     }));
 
+  describe(`#verify()`, () => {
+    it(`registers a new user`, () => spark.user.verify({email: `Collabctg+spark-js-sdk-${uuid.v4()}@gmail.com`})
+      .then((res) => {
+        console.log(res);
+        assert.property(res, `showPasswordPage`);
+        assert.isFalse(res.showPasswordPage);
+
+        assert.isFalse(res.showConfirmationCodePage);
+        assert.isFalse(res.showPasswordPage);
+        assert.isTrue(res.isUserCreated);
+        assert.isFalse(res.isSSOUser);
+        assert.isTrue(res.newUserSignUp);
+      }));
+
+    it(`leaves email address validation up to Atlas`, () => assert.isRejected(spark.user.register({email: `not an email address`}))
+      .then((res) => assert.statusCode(res, 400)));
+  });
+
+  // This test relies on setting a specific user agent, so it doesn't work in
+  // browsers
+  nodeOnly(describe)(`#activate()`, () => {
+    it(`completes the mobile signup process`, () => spark.request({
+      service: `atlas`,
+      resource: `users/email/verify`,
+      method: `POST`,
+      body: {
+        deviceName: `DESKTOP`,
+        deviceId: `not a mobile device`,
+        pushId: `not a mobile device`,
+        email: `Collabctg+spark-js-sdk-${uuid.v4()}@gmail.com`
+      },
+      requiresClientCredentials: true,
+      headers: {
+        'user-agent': `wx2-android`
```

Comment:
You're using this user-agent to test mobile, but providing junk push and device ids above because the notifications don't have anywhere to go, (and aren't necessary for this test) right?